bin/reference_data.py

```diff
@@ -469,6 +469,51 @@ def download_data(self):
             self.reference_variation = vcf
 
 
+def upload_species_without_genome(species_name):
+    """
+    This function only adds a new species without the need for a new genome.
+    An existing genome will be associated with the new species.
+    """
+    logger = logging_default.get_logger('SpeciesWithoutGenome')
+
+    scientific_name = ncbi.get_species_name(species_name)
+    if not scientific_name:
+        raise EGCGError('Species %s could not be resolved in NCBI please check the spelling.', species_name)
+
+    species = rest_communication.get_document('species', where={'name': scientific_name})
+    if species:
+        logger.error('Species %s already exist in the API', species_name)
+        return 1
+
+    genome_version = input(
+        "Enter genome version to use as the default genome for %s " % scientific_name
+    )
+    genome_data = rest_communication.get_document('genomes', where={'assembly_name': genome_version})
+    if not genome_data:
+        logger.error('Genome %s does exist in the API. Add it separately with its own species', genome_version)
+        return 1
```

Comment:
What if you return different error numbers, to make it easier to trace an error should it occur?

Reply:
The error logging already does this. That said, my preference would be to raise an exception as per the rest of the program and in line 481. If we do want to use return values, we'd want to capture the result in the call at line 536, and return 0 at the end of this function